readme-translations/README.br.md

```diff
@@ -0,0 +1,200 @@
+# Spring Cloud Example
+
+[![Build Status](https://travis-ci.com/naXa777/spring-cloud-example.svg?branch=master&style=flat)](https://travis-ci.com/naXa777/spring-cloud-example)
+
+Esse playground é para aprender [Spring Cloud](https://cloud.spring.io/) e [Kotlin](https://kotlinlang.org/) através de exemplos.
+
+Leia em outros idiomas: 
+* [На русском](../README.md) - Russo
+* [Bahasa Indonesia](README.id.md) - Indonésio
+* [In het Nederlands](README.nl.md) - Holândes
```

Comment:
Do not forget to add link to English readme

Reply:
done :)

Reply:
nice :) but what about crosslinks to this file from other readmes?